extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/devmode/console/DevConsoleProcessor.java

```diff
@@ -482,4 +490,29 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
             });
         }
     }
+
+    public static class JavaDocResolver implements ValueResolver {
+
+        private final Pattern codePattern = Pattern.compile("(\\{@code )([^}]+)(\\})");
+        private final Pattern linkPattern = Pattern.compile("(\\{@link )([^}]+)(\\})");
+
+        @Override
+        public boolean appliesTo(EvalContext context) {
+            return context.getBase() instanceof String && context.getName().equals("fmtJavadoc");
+        }
+
+        @Override
+        public CompletionStage<Object> resolve(EvalContext context) {
+            String val = context.getBase().toString();
+            // Replace {@code} and {@link}
+            val = codePattern.matcher(val).replaceAll("<code>$2</code>");
+            val = linkPattern.matcher(val).replaceAll("<code>$2</code>");
```

Comment:
```suggestion
            val = linkPattern.matcher(val).replaceAll("<a href="$2">$2</a>");
```
And I would add a test :)

Reply:
I don't think this would work. The value we get is not evaluated, i.e. for `{@link AutoInjectAnnotationBuildItem}` this code would render `<a href="AutoInjectAnnotationBuildItem">AutoInjectAnnotationBuildItem</a>`...

Reply:
Ah yes, you're right. Sorry about the noise.